main/src/addins/MonoDevelop.Debugger/MonoDevelop.Debugger.PerfTests/MonoDevelop.Debugger.PerfTests.csproj

```
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\..\..\..\MonoDevelop.props" />
+  <Import Project="$(ReferencesGtk)" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{785F775C-9CA0-4411-86E6-B89D89F3E2ED}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <RootNamespace>MonoDevelop.Debugger.PerfTests</RootNamespace>
+    <AssemblyName>MonoDevelop.Debugger.PerfTests</AssemblyName>
+    <TargetFrameworkVersion>$(MDFrameworkVersion)</TargetFrameworkVersion>
+    <TestRunnerCommand>..\..\..\..\build\bin\mdtool.exe</TestRunnerCommand>
+    <TestRunnerArgs>run-md-tests</TestRunnerArgs>
+    <OutputPath>..\..\..\build\tests</OutputPath>
```

@jstedfast okay, you can set those 2 back to `<Private>false</Private>`

```suggestion
    <OutputPath>..\..\..\..\build\tests</OutputPath>
```